feat(EOL): Added initial EOL content that's active and announced  in Q2 (#18232)
newrelic/docs-website: src/content/eol/2024/08/eol-08-01-24.md

> ---

Reviewer: we should probably delete your new WN post and redirect it to this page to avoid duplicative announcements

Author: I'll do that once this goes live.

title: "EOL for Google Core Web Vital FID metric"
summary: "On September 9, 2024, Google is deprecating the FID metric, in favor of Interaction to Next Paint (“INP”)."
publishDate: '2024-08-01'
eolEffectiveDate: '2024-09-09'
---

> On September 9, 2024, [Google is deprecating the FID metric](https://developers.google.com/search/blog/2023/05/introducing-inp), in favor of Interaction to Next Paint (“INP”). Since New Relic uses certain libraries and APIs that leverage the FID metric, you will need to take the actions below before September 9, 2024 to avoid inaccurate or unexpected behavior with dashboards, alerts, and service level reporting. 

Reviewer: Here and elsewhere in this doc, I suggest saying `custom dashboards` to avoid the impression that built-in/curated dashboards in the Browser UI will break.

## What you need to do

To avoid inaccurate alerts, charts, and service levels, you must make the following changes **before September 9, 2024**:

> 1. Edit all **custom dashboards** using FID from the PageViewTiming event and replace with INP. 

Reviewer: Here and elsewhere, `PageViewTiming` should be in code since it's an NR event

2. Edit any NRQL **alert conditions** using FID from the PageViewTiming event and replace with INP. 

> 3. Edit any **Service Levels** using FID from the PageViewTiming event and use INP.

Reviewer: ```suggestion
3. Edit any **Service Levels** using FID from the PageViewTiming event and replace with INP.
```

We use `replace with` on the other OL items

> Resources for how you can identify which dashboards, alerts, and service levels use the FID metric can be found below. 

Reviewer: ```suggestion
See the resources below for more information on how to identify which dashboards, alerts, and service levels use the FID metric. 
```
